### ISSUE TITLE
Deprecation warning in Angular console re Job Chat - DOMNodeInserted

### ISSUE THREAD

**Issue by samschlicht** [talentbeyondboundaries/tbbtalentv2#724]
Logging this for investigation. The deprecated event type warning is produced when we click on a specific job chat, as pictured.

FYI for whoever looks into this: it was happening _before_ my emoji branch was merged, so you can ignore that part of the code to begin with.

<img width="1455" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBNEM4QXc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--fa938d36d248b50e244a3384eae0630a91a8f2bb/Screenshot%202024-03-07%20at%209.34.23%E2%80%AFam.png" alt="Screenshot 2024-03-07 at 9.34.23 am.png" />

<img width="1424" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBNEc4QXc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--c1a673e7733a5b9bce4739ec4bbdf6ba78c266d6/Screenshot%202024-03-07%20at%209.34.44%E2%80%AFam.png" alt="Screenshot 2024-03-07 at 9.34.44 am.png" />

**Comment by samschlicht** [comment]
This error is coming from Quill:

<img width="1138" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBOGtVQ0E9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--e862151033362bd32f1808b1feffe03c2a37f09c/Screenshot%202024-12-10%20at%2012.10.39%E2%80%AFpm.png" alt="Screenshot 2024-12-10 at 12.10.39 pm.png" />


<img width="971" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBOG9VQ0E9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--d2e9e107fc1a44c530f9d11a02de221dd2051979/Screenshot%202024-12-10%20at%2012.11.36%E2%80%AFpm.png" alt="Screenshot 2024-12-10 at 12.11.36 pm.png" />

**Comment by samschlicht** [comment]
The issue is [well documented on their GitHub](https://github.com/slab/quill/issues/3806), and fixed in V2.

Unfortunately, if using ngx-quill (as we are), Quill V2 needs Angular 17. We're on 16, so this will need to wait till our next Angular upgrade. 

However, looking at the code and its accompanying comment, it is a little hacky and doesn't appear to be doing much. It's no longer recognised by my version of Chrome but I've had a good play around with the Quill editor in TC Test and it's working totally fine.

So, to summarise: seeing this in the console is a small but insignificant niggle that we can eliminate when we move to Angular 17, should we choose also to upgrade Quill at that point — but this issue alone wouldn't necessarily be sufficient cause to do so. 

UPDATE: decided in team meet to leave this here as an aide memoire marked for v2.4.0 when we can make the call about upgrading Quill (we won't be upgrading to Angular 17 in this cycle).

**Comment by samschlicht** [comment]
Confirming that this warning is no longer appearing following the upgrade to Angular 17, as predicted. 😎👍
